### PR TITLE
switch to using rpms for adding files

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: mediawiki123-apb/Dockerfile [hunk 1]
```diff
@@ -23,8 +23,6 @@ ZGlhd2lraV9kYl9zY2hlbWEKICAtIG1lZGlhd2lraV9zaXRlX25hbWUKICAtIG1lZGlhd2lraV9z\
 aXRlX2xhbmcKICAtIG1lZGlhd2lraV9hZG1pbl91c2VyCiAgLSBtZWRpYXdpa2lfYWRtaW5fcGFz\
 cwo="
 
-
-ADD playbooks /opt/apb/actions
-ADD roles /opt/ansible/roles
+RUN yum -y install mediawiki-apb-role && yum clean all
```
Comment (fabianvf) [inline mediawiki123-apb/Dockerfile:26]: Could you create a `Dockerfile-dev` with the old behavior to make development/iteration easier for these?

File: rhscl-postgresql-apb/Dockerfile [hunk 2]
```diff
@@ -22,7 +22,7 @@ CiAgICAtIHRpdGxlOiBQb3N0Z3JlU1FMIFZlcnNpb24KICAgICAgdHlwZTogZW51bQogICAgICBk\
 ZWZhdWx0OiA5LjUKICAgICAgZW51bTogWyI5LjUiLCAiOS40Il0KcmVxdWlyZWQ6CiAgLSBwb3N0\
 Z3Jlc3FsX2RhdGFiYXNlCiAgLSBwb3N0Z3Jlc3FsX3VzZXIKICAtIHBvc3RncmVzcWxfdmVyc2lv\
 bgo="
-dGVyczoge30K"
 
-COPY roles /opt/ansible/roles
-COPY playbooks/* /opt/apb/actions/
+RUN yum -y install postgresql-apb-role && yum clean all
```
Comment (fabianvf) [inline rhscl-postgresql-apb/Dockerfile:26]: same as above

**Review by fabianvf — APPROVED** [review]

File: mediawiki123-apb/Dockerfile (hunk 1 above)
Comment (jmontleon) [reply mediawiki123-apb/Dockerfile:26]: sure

File: rhscl-postgresql-apb/Dockerfile (hunk 2 above)
Comment (jmontleon) [reply rhscl-postgresql-apb/Dockerfile:26]: done

**Pull request merged**